gh-pr-context.py

```
@@ -88,16 +88,79 @@ def gh_api_jq(endpoint, jq_filter):
+            os.environ.pop("GIT_WORK_TREE", None)
+
+
 def check_deps():
     from shutil import which
     git_bin = _git_cmd()[0]
     gh_bin = _gh_cmd()[0]
     for label, binary in (("git", git_bin), ("gh", gh_bin)):
         if not which(binary):
             die(f"{label} is required but not found on PATH")
+    _setup_git_env()
     out, rc = run_cmd(_git_cmd() + ["rev-parse", "--git-dir"])
```

Ruff (0.15.15)

[warning] Unpacked variable `out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)